macros/meta/get_monitored.sql

```diff
@@ -0,0 +1,92 @@
+{% macro pub_monitored_from_graph() %}
+    {% set monitored = [] %}
+    {% set both = []%}
+    {% do both.extend(graph.nodes.values()) %}
+    {% do both.extend(graph.sources.values()) %}
+
+    {% for el in both %}
+        {% if el.resource_type in ['model', 'seed', 'source'] %}
+            {% if el.config.get('re_data_monitored') %}
+            
+                {% do monitored.append({
+                    'name': re_data.name_in_db(el.name),
+                    'schema': re_data.name_in_db(el.schema),
+                    'database': re_data.name_in_db(el.database),
+                    'time_filter': el.config.get('re_data_time_filter', none),
+                    'metrics': re_data.metrics_in_db(el.config.get('re_data_metrics', {})),
+                    'columns': re_data.columns_in_db(el.config.get('re_data_columns', []))
+                    })
+                %}
+            {% endif %}
+        {% endif %}
+    {% endfor %}
+
+    {{ return(monitored) }}
+{% endmacro %}
```

Comment:
According to [dbt](https://docs.getdbt.com/reference/source-configs),
```
Unlike other resource types, sources do not yet support a config property. It is not possible to (re)define source configs hierarchically across multiple yaml files.
```

We won't be able to pass `re_data_monitored` in this case to the sources definition, I tried and it didn't show up. Did you somehow get this working?

Reply:
Yeah, that's true. What works is configuration is dbt_project.yml sources config. Check out https://docs.getre.io/feature-dbt-config-support/docs/reference/config and let me know if those are clear ;)

Reply:
Great! That's very clear.